eng/verify-nupkgs-exe.ps1

```diff
@@ -0,0 +1,90 @@
+$exclusions = @{
+    "CodeCoverage\CodeCoverage.exe"                = "x86"
+    "Dynamic Code Coverage Tools\CodeCoverage.exe" = "x86"
+    "amd64\CodeCoverage.exe"                       = "x64"
+
+    "IntelliTrace.exe"                             = "x86"
+    "ProcessSnapshotCleanup.exe"                   = "x86-64"
+    "TDEnvCleanup.exe"                             = "x86"
+
+    "TestPlatform\SettingsMigrator.exe"            = "x86"
+
+    "dump\DumpMinitool.exe"                        = "x86-64"
+
+    "QTAgent32.exe"                                = "x86"
+    "QTAgent32_35.exe"                             = "x86"
+    "QTAgent32_40.exe"                             = "x86"
+    "QTDCAgent32.exe"                              = "x86"
+
+    "V1\VSTestVideoRecorder.exe"                   = "x86"
+    "VideoRecorder\VSTestVideoRecorder.exe"        = "x86"
+}
+
+$errs = @()
+Get-ChildItem  S:\p\vstest3\artifacts\packages\Debug\Shipping -Filter *.exe -Recurse -Force | ForEach-Object {
+    $m = & "C:\Program Files\Microsoft Visual Studio\2022\IntPreview\VC\Tools\MSVC\14.38.32919\bin\HostX86\x86\dumpbin.exe" /headers $_.FullName | Select-String "machine \((.*)\)"
```

Comment:
Can we use some env var for the beginning of the path so that this can handle have VS installed on a different path?